src/Containers/CardCarousel/CardCarousel.tsx

```diff
@@ -0,0 +1,388 @@
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import {
+    Calendar,
+    CreditCard,
+    X,
+    Hourglass,
+    PersonFill,
+    ArrowRepeat,
+} from '@styled-icons/bootstrap';
+
+export interface CardCarouselProps {
+    copyHeader: string;
+    copyBody: string;
+    cardTitle: string;
+    cardSubtitle: string;
+}
+
+export const CardCarousel: React.FC<CardCarouselProps> = ({
+    copyHeader,
+    copyBody,
+    cardTitle,
+    cardSubtitle,
+}) => (
+    <>
+        <Section>
+            <Copy>
+                <header>
+                    <h1>{copyHeader}</h1>
+                </header>
+                <p>{copyBody}</p>
+            </Copy>
+            <Carousel>
+                <CarouselCard>
+                    <CardHead>
+                        <CreditCardIcon size="35" />
+                    </CardHead>
+                    <CardBody>
+                        <Title>{cardTitle}</Title>
+                        <Subtitle>{cardSubtitle}</Subtitle>
+                    </CardBody>
+                </CarouselCard>
+                <CarouselCard>
+                    <CardHead>
+                        <XIcon size="35" />
+                    </CardHead>
+                    <CardBody>
+                        <Title>{cardTitle}</Title>
```

Comment:
Shouldn't each one have its own title?
Don't you need to map through N ARRAY of objects to display the number of cards in the carousel?